### PR TITLE
[ mysql-PositiveTechnologies][imporve] revise Positive Technologies‘s README.md

### PR DESCRIPTION
**Pull request by caicancai**
Positive Technologies currently seems to have mysql 8.0, can modify the README file, tell users

### CODE REVIEW

**Comment by caicancai** [comment]
@KvanTTT

File: sql/mysql/README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 This folder contains MySQL grammars from different authors. See the specific sub folders for more details:
 
 * [Oracle](Oracle) - A complete grammar, mostly derived from the MySQL server grammar (5.6, 5.7, 8.0), as used in MySQL Workbench
-* [Positive Technologies](Positive-Technologies) - A grammar created from the official documentation (5.6, 5.7)
+* [Positive Technologies](Positive-Technologies) - A grammar created from the official documentation (5.6, 5.7,8.0)
```
Comment (KvanTTT) [inline sql/mysql/README.md:6]: Please add space after comma

```suggestion
* [Positive Technologies](Positive-Technologies) - A grammar created from the official documentation (5.6, 5.7, 8.0)
```

File: sql/mysql/Positive-Technologies/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Positive Technologies MySQL grammar
 
-An ANTLR4 grammar for MySQL based on version 5.6 and 5.7 of 
+An ANTLR4 grammar for MySQL based on version 5.6 , 5.7 and 8.0 of
```
Comment (KvanTTT) [inline sql/mysql/Positive-Technologies/README.md:3]:
```suggestion
An ANTLR4 grammar for MySQL based on version 5.6, 5.7 and 8.0 of 
```

File: sql/mysql/Positive-Technologies/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Positive Technologies MySQL grammar
 
-An ANTLR4 grammar for MySQL based on version 5.6 and 5.7 of 
+An ANTLR4 grammar for MySQL based on version 5.6 , 5.7 and 8.0 of 
 [http://dev.mysql.com/doc/refman/5.6/en/](http://dev.mysql.com/doc/refman/5.6/en/).
-[http://dev.mysql.com/doc/refman/5.7/en/](http://dev.mysql.com/doc/refman/5.7/en/).
+[http://dev.mysql.com/doc/refman/5.7/en/](http://dev.mysql.com/doc/refman/5.7/en/).                                       [http://dev.mysql.com/doc/refman/8.0/en/](http://dev.mysql.com/doc/refman/8.0/en/).
```
Comment (KvanTTT) [inline sql/mysql/Positive-Technologies/README.md:5]: Line break is missing

```suggestion
[http://dev.mysql.com/doc/refman/5.7/en/](http://dev.mysql.com/doc/refman/5.7/en/).
[http://dev.mysql.com/doc/refman/8.0/en/](http://dev.mysql.com/doc/refman/8.0/en/).
```

**Review by KvanTTT — CHANGES_REQUESTED** [review]

**Comment by caicancai** [comment]
@KvanTTT  fix

**Review by KvanTTT — APPROVED** [review]

**Pull request merged**